docs/components.md

```diff
@@ -43,14 +43,14 @@ The `onDestroy` method can be overridden to run code before the component is rem
 
 There are also other implementations:
 
-* The `AnimationComponent` takes an `Animation` object and renders a cyclic animated sprite (more details about Animations [here](/doc/images.md#Animation))
+* The `AnimationComponent` takes an `Animation` object and renders a cyclic animated sprite (more details about Animations [here](./images.md#Animation))
 * The `SvgComponent` takes an `Svg` object and renders the SVG on the game
 * The `ParallaxComponent` can render a parallax background with several frames
 * The `Box2DComponent`, that has a physics engine built-in (using the [Box2D](https://github.com/google/box2d.dart) port for Dart)
 
 ## Animation Component
 
-This component uses an instance of the [Animation](/doc/images.md#Animation) class to represent a Component that has a sprite that runs a single cyclic animation.
+This component uses an instance of the [Animation](./images.md#Animation) class to represent a Component that has a sprite that runs a single cyclic animation.
```

Comment:
we probably need to keep the old format, as there is a script that reads this files and generate the site based on it and I don't know if this will work.

@luanpotter what do you think? since you made the script?

Reply:
Where does this script reside? :)

Reply:
It is on the flame engine site repository, under the folder generate docs, I would get you a link, but the GH mobile app still has lots to improve hahhahaha

Reply:
Haha I can probably find it from that description.

Reply:
Reverted to use the full paths

Reply:
https://github.com/flame-engine/flame-engine-site/blob/master/cmds/generate_docs.sh

yes we do need the full path. also changing the folder name might require a second PR on the other repo

Reply:
Did a PR for that repository too:
https://github.com/flame-engine/flame-engine-site/pull/1